content/doc/book/updating/_update-wizard.adoc

```diff
@@ -0,0 +1,70 @@
+////
+This file is only meant to be included as a snippet in other documents.
+There is a version of this file for the general 'Updating Jenkins' page (index.adoc).
+This file is for the index.adoc page used in the general 'Updating Jenkins' page.
```

Comment:
This file is not included on the index page, so it does not appear at this time. It is included in other updating pages, but if it is general and should be in the index, it should be added there too.

Reply:
Actually i followed the [content/doc/book/installing/index.adoc](https://github.com/jenkins-infra/jenkins.io/blob/master/content/doc/book/installing/index.adoc) for the file structure that's why this line is there and it should be removed from there.